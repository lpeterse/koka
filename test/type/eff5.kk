// test if we keep effects polymorph
// ok     : forall<a,e> (f: (x: a) -> e a) -> ((x: a) -> e a)
fun ok(f) = choose(id,f)
