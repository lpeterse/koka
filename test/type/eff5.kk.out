ok     : forall<a,e> (f : (x : a) -> e a) -> ndet ((x : a) -> e a)
